content/12.privacy.md

```diff
@@ -1,12 +1,28 @@
-## Tip 10: Don't share models trained on sensitive data {#privacy}
+## Tip 10: Actively consider the ethical implications of your work {#privacy}
 
-Practitioners may encounter datasets that cannot be shared, such as ones for which there would be significant ethical or legal issues associated with their release [@doi:10.1371/journal.pcbi.1005399].
+While deep learning continues to be a powerful, transformative tool within life sciences research—spanning basic biology and pre-clinical science to varied translational approaches and clinical studies—it is important to comment on some ethics—related considerations.
+For instance, despite the fact that deep learning methods are helping to increase medical efficiency through improved diagnostic capability and risk assessment, certain biases may be inadvertently introduced into models related to patient age, race, and gender [@doi:10.1002/hast.977]; as previously mentioned, deep learning practitioners may make use of datasets not representative of diverse populations and patient characteristics [@doi:10.1377/hlthaff.2014.0048], thereby contributing to these problems (please refer to Tip 4).
+
+Therefore, it is important to think thoroughly and cautiously about deep learning applications and its potential impact to persons and society—mindful of possible harms, injuries, injustices, and other types of wrongdoings.
+At a minimum, practitioners must ensure that, wherever relevant, their life sciences projects are fully compliant with local research governance/approval policies, legal requirements, institutional review board (IRB) policies, and any other relevant bodies and their standards.
+Moreover, we offer below three tangible, action-oriented recommendations to further empower and enrichen deep learning researchers.
+
+First, similarly to how certain teams keep a project-specific or programming-related issue tracker detailing known bugs and other technical issues, practitioners should get into the habit of keeping an active _ethics register_.
+Teams can institute what Rachel Thomas of the Center for Applied Data Ethics at the University of San Francisco terms “ethical risk sweeps.”
```

Comment:
I think this section could be strengthened by eliminating the direct quotations and adding citations to the literature referred to in the popular news article rather than the article itself.

Reply:
If the popular news articles are referencing peer-reviewed articles regarding their main points, I agree that we should reference those instead. They will likely have a permanent DOI, too, whereas popular news articles may be deleted or moved in the future and may not be accessible in the future

Reply:
Noted and we can make this change once the paper is under review

Reply:
yeah that sounds good to me